internal/tiltfile/docker_compose_test.go

```diff
@@ -20,24 +20,24 @@ func TestParseConfigPreservesServiceOrder(t *testing.T) {
 services:
   a:
     image: imga
+    depends_on: [b]
   b:
     image: imgb
+    depends_on: [c]
   c:
     image: imgc
+    depends_on: [d, e, f]
   d:
     image: imgd
+    depends_on: [f, e]
   e:
     image: imge
+    depends_on: [f]
   f:
-    image: imgf`
-
-	var servicesOutput = `f
-e
-d
-c
-b
-a`
-	services := f.parse(output, servicesOutput)
+    image: imgf
+`
+
+	services := f.parse(output)
```

Comment:
Implicit YAML ordering was never a guarantee of Compose - it implicitly worked in v1 due to Python parsing, but is totally gone in v2 -- current Tilt + Compose v2 is actually broken in this respect right now.

Tilt will launch services in the same order as Compose v2 now, which will respect dependencies but otherwise be alphabetical.